packages/contracts-bedrock/deploy-config/devnetL1-template.json

```diff
@@ -40,10 +40,12 @@
   "governanceTokenName": "Optimism",
   "governanceTokenOwner": "0xa0Ee7A142d267C1f36714E4a8F75612F20a79720",
   "eip1559Denominator": 50,
+  "eip1559DenominatorPostCanyon": 250,
```

Comment:
`PostCanyon` vs just `Canyon` in the name?

Reply:
Yea, I've been thinking about changing it. Right now it's post canyon. Canyon could be good. I was also thinking about calling it `eip1559DenominatorV2`